_includes/auth/private-public-key-pairs-for-tiny-cloud-services.md

```diff
@@ -1,9 +1,21 @@
-## Private/public key pairs for {{site.cloudname}} services
+## Add a public key to the {{site.cloudname}} API key
 
-{{site.cloudname}} services tokens use public/private RSA key pairs. The {{site.cloudname}} services only store the public key, allowing developers to have full control over the authentication.
+The {{pluginname}} Server requires a _public_ key generated from the same _private_ key that will be used on your JSON Web Token (JWT) Provider server. The public key(s) stored on the {{pluginname}} Server are used to ensure that content is sent by authorized users.
 
-The private/public key pair can be created on your [{{site.accountpage}} page]({{site.accountpageurl}}), however the public key is only stored on the {{site.accountpage}}. The private key should be downloaded and stored in your backend.
+There are two methods for generating and adding a public key to your API key:
 
-> **Important**: Keep the private key secure. Do not commit files containing the key to public repositories or websites.
+1. The secure keypair generator at [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/) (recommend).
+1. Generate a keypair locally and add the _public_ key to [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/).
```

Comment:
```suggestion
1. The secure key pair generator at [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/) (recommend).
1. Generate a key pair locally and add the _public_ key to [{{site.accountpage}} - JWT Keys]({{site.accountpageurl}}/jwt/).
```
`keypair` isn't a word and looking at other terms used elsewhere it's always 2 words.